### PR TITLE
22725 Remove inputAndSelection dialog

### PR DESCRIPTION
**Pull request by d4hines**
refactor: #id [22725]

[Kanban link](https://chartiq.kanbanize.com/ctrl_board/18/cards/22725/details)

**Description of change**
Deletes the unused inputAndSelection dialog component.

This component was part of a workspace feature that was never fully realized.

### CODE REVIEW

File: src-built-in/components/workspaceManagementMenu/src/stores/workspaceManagementMenuStore.js [hunk 1]
```diff
@@ -462,7 +462,7 @@ Actions = {
 			showNegativeButton: false
 		};
 
-		Actions.spawnDialog("singleInput", dialogParams, onUserInput);
+		Actions.spawnDialog("inputAndSelection", dialogParams, onUserInput);
```
Comment (zona-ciq) [inline src-built-in/components/workspaceManagementMenu/src/stores/workspaceManagementMenuStore.js]: Why did you change this into "inputAndSelection" since we're removing it?

Comment (d4hines) [reply src-built-in/components/workspaceManagementMenu/src/stores/workspaceManagementMenuStore.js]: This was a mistake! Good catch!

Comment (zona-ciq) [reply src-built-in/components/workspaceManagementMenu/src/stores/workspaceManagementMenuStore.js]: I think there is another one here: https://github.com/ChartIQ/finsemble-seed/pull/618/files#diff-db4c8003ad6f961267ff893ab63305e7R492

**Comment by d4hines** [comment]
@meredith-ciq, that's my bad. I did not mean to commit those changes. Will fix, re-test, and let you know.

**Comment by meredith-ciq** [comment]
So Are you also wrapping singleinputdialog here too? Or did you branch off a different base?

**Comment by d4hines** [comment]
@meredith-ciq, that's a different PR.

**Comment by d4hines** [comment]
Oh, I see what you're saying @meredith-ciq! I'll remove those commits.

File: src-built-in/components/workspaceManagementMenu/src/stores/workspaceManagementMenuStore.js (hunk 1 above)
Comment (d4hines) [reply src-built-in/components/workspaceManagementMenu/src/stores/workspaceManagementMenuStore.js]: Fixed.
